### DISCUSSION TITLE
In WPF, when Popup opened, the webview2 control would lost focus.

### DISCUSSION THREAD

**Discussion by wslsj888** [MicrosoftEdge/WebView2Feedback#3884] [Q&A]
such as title, the js events like 'mousemove' would not be called when the popup opened.

**Comment by champnic** [comment]
Hey @wslsj888 - when you say "popup" could you add some detail on how the popup is being shown? Is this a web popup (like from `window.open()`) or a popup form you are creating in WPF?

**Comment by wslsj888** [comment]
> Hey @wslsj888 - when you say "popup" could you add some detail on how the popup is being shown? Is this a web popup (like from `window.open()`) or a popup form you are creating in WPF?

sorry, It is a wpf's popup.
web -> js -> C# -> open wpf's popup.

**Comment by champnic** [comment]
If you are opening a new WPF popup it makes sense that it may try to take focus. If you need to move focus back to the WebView2 control you can do so using WebView2.Focus().

**Comment by wslsj888** [comment]
> If you are opening a new WPF popup it makes sense that it may try to take focus. If you need to move focus back to the WebView2 control you can do so using WebView2.Focus().

I have tried WebView2.Focus() before. It does'n work.
maybe it's normal, so I use wpf's Window to instead the Popup. a new window would not let the webview2 control lost focus.

**Comment by champnic** [comment]
Do you have a sample app you would be able to share the demonstrates the broken behavior?
